### PR TITLE
fixed tutorial

### PR DESCRIPTION
**Pull request by TezRomacH**
## Description

Just fix some typos

## Related Issue

N/A

## Type of Change

- [x] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have read the [Code of Conduct](https://github.com/catalyst-team/catalyst/blob/master/CODE_OF_CONDUCT.md) document.
- [ ] I have read the [Contributing](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md) guide.
- [ ] I have checked the code-style using `make check-style`.
- [ ] I have written the docstring in Google format for all the methods and classes that I used.
- [ ] I have checked the docs using `make check-docs`.

### CODE REVIEW

File: examples/notebooks/segmentation-tutorial.ipynb [hunk 1]
```diff
@@ -827,7 +829,7 @@
    "outputs": [],
    "source": [
     "if is_alchemy_used:\n",
-    "    from catalyst.dl.runner import SupervisedAlchemyRunner as SupervisedRunner\n",
+    "    from catalyst.contrib.runner.alchemy import SupervisedAlchemyRunner as SupervisedRunner\n",
```
Comment (Scitator) [inline examples/notebooks/segmentation-tutorial.ipynb:832]: you don't need this, cause dl import everything from contrib already

File: requirements/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Framework must-haves
 numpy>=1.16.4
 torch>=1.0.0
+Pillow==6.2.2
```
Comment (Scitator) [inline requirements/requirements.txt:4]: for torchvision compatability, you need Pillow<7
we have already solve this issue in another PR

File: examples/notebooks/segmentation-tutorial.ipynb (hunk 1 above)
Comment (TezRomacH) [reply examples/notebooks/segmentation-tutorial.ipynb:832]: no, the current version in master throws an error. u can try it by yourself

File: requirements/requirements.txt (hunk 2 above)
Comment (TezRomacH) [reply requirements/requirements.txt:4]: I know, but without this line, I couldn't pass the CI

Comment (Scitator) [reply requirements/requirements.txt:4]: please merge with master, we have already fixed this issue

File: examples/notebooks/segmentation-tutorial.ipynb (hunk 1 above)
Comment (Scitator) [reply examples/notebooks/segmentation-tutorial.ipynb:832]: not reproduced

**Comment by Scitator** [comment]
hi, 
How does the PR going?

**Comment by Scitator** [comment]
looks like resolved :)

**Pull request closed without merging**